source/frontend/src/features/leases/detail/LeasePages/tenant/AddLeaseTenantContainer.tsx

```diff
@@ -91,26 +102,22 @@ export const AddLeaseTenantContainer: React.FunctionComponent<
   };
 
   return (
-    <>
-      <AddLeaseTenantForm
-        initialValues={{ ...defaultFormLease, ...apiLeaseToFormLease(lease as ILease) }}
-        selectedTenants={
-          selectedTenants.length
-            ? selectedTenants
-            : apiLeaseToFormLease(lease as ILease)?.tenants ?? []
-        }
-        setSelectedTenants={setSelectedTenantsWithPersonData}
-        onSubmit={onSubmit}
-        formikRef={formikRef}
-      >
-        {children}
-      </AddLeaseTenantForm>
-      <PrimaryContactWarningModal
-        saveCallback={handleSubmit}
-        onCancel={() => setHandleSubmit(undefined)}
-        lease={formikRef?.current?.values}
-      />
-    </>
+    <View
```

Comment:
refactored into new dependency injected version since it simplifies unit testing.